desktop/core/requirements.txt

```diff
@@ -64,3 +63,4 @@ sqlparse==0.4.1
 tablib==0.13.0
 thrift==0.13.0
 thrift-sasl==0.4.2
+git+git://github.com/agl29/django-babel.git
```

Comment:
can we do this @romainr?

Reply:
:+1: 
And we could move it to http://github.com/gethue at the end of Django 3 if the lib is still needed